agenta-backend/agenta_backend/models/db_models.py

```diff
@@ -1,303 +1,470 @@
+    user = relationship("UserDB")
+    name = Column(String)
+    evaluator_key = Column(String)
+    settings_values = Column(JSONB, default=dict)
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+
+class HumanEvaluationDB(Base):
+    __tablename__ = "human_evaluations"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    app_id = Column(UUID(as_uuid=True), ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    status = Column(String)
+    evaluation_type = Column(String)
+    variant_id = Column(UUID(as_uuid=True), ForeignKey("app_variants.id"))
+    variant = relationship("AppVariantDB")
+    variant_revision_id = Column(
+        UUID(as_uuid=True), ForeignKey("app_variant_revisions.id")
+    )
+    variant_revision = relationship("AppVariantRevisionsDB")
+    testset_id = Column(UUID(as_uuid=True), ForeignKey("testsets.id"))
+    testset = relationship("TestSetDB")
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+
+
+class HumanEvaluationScenarioDB(Base):
+    __tablename__ = "human_evaluations_scenarios"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    evaluation_id = Column(UUID(as_uuid=True), ForeignKey("human_evaluations.id"))
+    evaluation = relationship("HumanEvaluationDB")
+    inputs = Column(JSONB)  # List of HumanEvaluationScenarioInput
+    outputs = Column(JSONB)  # List of HumanEvaluationScenarioOutput
+    vote = Column(String)
+    score = Column(JSONB)
+    correct_answer = Column(String)
+    created_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    updated_at = Column(
+        DateTime(timezone=True), default=lambda: datetime.now(timezone.utc)
+    )
+    is_pinned = Column(Boolean)
+    note = Column(String)
+
+
+class EvaluationDB(Base):
+    __tablename__ = "evaluations"
+
+    id = Column(
+        UUID(as_uuid=True),
+        primary_key=True,
+        default=uuid.uuid7,
+        unique=True,
+        nullable=False,
+    )
+    app_id = Column(UUID(as_uuid=True), ForeignKey("app_db.id"))
+    app = relationship("AppDB")
+    user_id = Column(UUID(as_uuid=True), ForeignKey("users.id"))
+    user = relationship("UserDB")
+    status = Column(JSONB)
```

Comment:
Same as [this](https://github.com/Agenta-AI/agenta/pull/1753/files#r1629550014).

Reply:
see the commented code